Resources/Prototypes/Entities/Objects/Devices/Circuitboards/computer.yml

```
@@ -114,11 +114,11 @@
   name: cargo bounty computer board
   description: A computer printed circuit board for a cargo bounty computer.
   components:
-  - type: Sprite
-    state: cpu_supply
-  - type: ComputerBoard
-    prototype: ComputerCargoBounty
-  - type: StaticPrice
+    - type: Sprite
+      state: cpu_supply
+    - type: ComputerBoard
+      prototype: ComputerCargoBounty
+    - type: StaticPrice
```

```suggestion
  - type: Sprite
    state: cpu_supply
  - type: ComputerBoard
    prototype: ComputerCargoBounty
  - type: StaticPrice
```
The previous indentation was correct.
See https://docs.spacestation14.com/en/general-development/codebase-info/conventions.html#yaml-conventions
Could you change it back and adjust it below as well? At least for the cases you are editing. Leave the other ones as they were to minimize merge conflicts.

---

Will correct. I should have checked style guide, I just went with the majority style.